WordPress/twentytwentytwo
# Add page block patterns

**Description**

Closes https://github.com/WordPress/twentytwentytwo/issues/19

This PR creates a blank page template that only adds the post-content block and the 3 page layout patterns from https://github.com/WordPress/twentytwentytwo/issues/19

**Testing Instructions** 

1. Create a new page
2. Select the Blank page template on the sidebar
3. Insert one of the new patterns under the `Twenty Twenty-Two Page layouts` category and check that everything works and looks as intended
